src/components/Badge/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Link } from "gatsby";
+import { TooltipWrapper, TooltipText } from "./Badge.styles";
+
+const Badge = ({ name, to, className, src, alt, firstChild }) => {
+  return (
+    <li>
```

Comment:
Hey @roopeshsn,
as we know that this is a `Badge` component, so having the UI only related to the badge would be better. the `li` tag is not relevant to the badge. Having it here kinda means that it'd be used in `ul` or `ol`, although we can use it without those list stage but then `li` would be better which would be useless.  
What do you think?

Reply:
Already Link component is used, if I am not wrong @thoughtlessmind

Reply:
Should I need to li tag alone?

Reply:
I meant that right now, the `Badge` component is very bound and specific to only certain use cases. Suppose you need to put a badge only, without a list and without a link, then again you need to create a component. This is called separation of concern. So, `Badge` component should be responsible for only adding the badge related things on the UI.